web-client/src/modules/index.ts

```diff
@@ -3,7 +3,7 @@ import { Module } from 'src/types/module';
 import Create from './create';
```

Comment:
Import rename based on dir change and added a TODO comment about looking at Posts more carefully.